Sources/Sentry/SentryProfiler.mm

```diff
@@ -254,7 +254,7 @@ - (SentryEnvelopeItem *)buildEnvelopeItemForTransaction:(SentryTransaction *)tra
     profile[@"device_physical_memory_bytes"] =
         [@(NSProcessInfo.processInfo.physicalMemory) stringValue];
     profile[@"environment"] = transaction.environment;
-    profile[@"platform"] = transaction.platform;
+    profile[@"platform"] = transaction.platform ?: @"production";
```

Comment:
This is the only change

The rest are formatting changes tacked on by pre-commit - how can we avoid master being unformatted?

Reply:
It would appear that the pre-commit format is different than what is committed by the CI code format job 🤔 See https://github.com/getsentry/sentry-cocoa/pull/2041/commits/20a7aa68ebed94bee3e1d643e238ca38fa7c2039